packages/roosterjs-editor-dom/lib/utils/highlightTableSelection.ts

```diff
@@ -0,0 +1,107 @@
+import findClosestElementAncestor from './findClosestElementAncestor';
+import getTagOfNode from './getTagOfNode';
+import safeInstanceOf from './safeInstanceOf';
+import VTable from '../table/VTable';
+import { IContentTraverser } from 'roosterjs-editor-types';
+
+/**
+ * Highlight the table cells when are in the selection range
+ * @param selection Wether a Selection from the document when browser is Firefox or a Content Traverser
+ */
+export const highlightTableSelection = (selection: Selection | IContentTraverser) => {
+    let firstTDSelected: HTMLTableCellElement;
+    let lastTDSelected: HTMLTableCellElement;
+    let table: HTMLTableElement;
+
+    if (selection instanceof Selection) {
```

Comment:
instanceof won't work correctly in projection popout.

Do not use suggest not use Selection as parameter